doc/en/user/source/extensions/importer/rest_reference.rst

```diff
@@ -913,6 +913,28 @@ Remaps a certain field to a given target data type
      - The "target" field type, as a fully qualified Java class name
 
 
+AttributeComputeTransform
+"""""""""""""""""""""""""
+
+Computes a new field based on an expression fed the other field values
+
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Optional
+     - Description
+   * - field
+     - N
+     - The name of the field to be computed
+   * - fieldType
+     - N
+     - The field type, as a fully qualified Java class name
```

Comment:
I would add an example, I guess for most users `java.lang.String` is not evident :stuck_out_tongue:

Reply:
Good point